js/libs/builder.js

```diff
@@ -832,6 +809,14 @@ var app = new Vue({
     'settings.dataStore': function(value) {
       this.showExtraAdd = value.indexOf('dataSource') > -1;
       this.showExtraEdit = value.indexOf('editDataSource') > -1;
```

Comment:
what does showExtraEdit mean? I also don't see it defined in the data object

Reply:
@squallstar If I understand it right it is showing an extra setting when we click on ` Update existing entries in a data source` checkbox.
![image](https://user-images.githubusercontent.com/53430352/93851514-ef77a880-fcb8-11ea-9f7b-38713d4c29fe.png)
Same thing for `showExtraAdd `.